kie-osgi/kie-karaf-features/pom.xml

```diff
@@ -82,6 +82,9 @@
     <karaf.version.org.apache.aries.jpa>1.0.1</karaf.version.org.apache.aries.jpa>
     <h2.version>1.4.178</h2.version>
 
+    <!-- Camel -->
+    <karaf.version.org.apache.camel>2.21.5</karaf.version.org.apache.camel>
```

Comment:
can we use standard property for versions like `version.<groupId>` ?

Reply:
I only used existing convention for properties used to filter values in feature.xml files. Please treat this PR as base for your work.

Reply:
ok, no big deal, but if we need to change that property version later in prod process this could be skipped due different structure, but we can merge it now to fix instabilities in tests